### PR TITLE
Ci transfer to pytest

### PR DESCRIPTION
**Pull request by nabaco**
is_terminal_state() is not passing tests.

### CODE REVIEW

File: envs/env_4inrow.py [hunk 1]
```diff
@@ -70,8 +70,9 @@ def available_moves(self, player):
 
     def is_terminal_state(self):
         status = True
-        if any(not cell for cell in self.board[0]) \
-                or not self.player_status(self.player1):
+        if any(cell == 0 for cell in self.board[0]) \
```
Comment (levzlotnik) [inline envs/env_4inrow.py]: This line says the following:  
* If there is an empty cell - return false
* if there isn't an empty cell - check:  
    - if one of the players has a non zero status - return true
    - else - return false

Now, if a player has won/lost, it's indeed a terminal state, no arguing there.  
However, there **could** be an empty cell and the state would still be terminal.  You should rewrite your logic, this is why your tests are failing.

**Review by levzlotnik — CHANGES_REQUESTED** [review]

**Review by David6114 — COMMENTED** [review]
OK

File: envs/env_4inrow.py [hunk 2]
```diff
@@ -69,11 +69,11 @@ def available_moves(self, player):
...
     def is_terminal_state(self):
-        status = True
-        if any(cell == 0 for cell in self.board[0]) \
-                or self.player_status(self.player1) == 0 \
-                or self.player_status(self.player2) == 0:
-            status = False
+        status = False
+        if all(cell for cell in self.board[0])\
+                or self.player_status(self.player1)\
+                or self.player_status(self.player1):
```
Comment (levzlotnik) [inline envs/env_4inrow.py]: Did you mean `self.player_status(self.player2)`?

**Review by levzlotnik — CHANGES_REQUESTED** [review]

File: envs/env_4inrow.py (hunk 2 above)
Comment (David6114) [reply envs/env_4inrow.py]: yes... just second...

Comment (David6114) [reply envs/env_4inrow.py]: sorry...

**Review by levzlotnik — APPROVED** [review]

**Pull request merged**